packages/strapi-admin/admin/src/translations/fr.json

```diff
@@ -55,6 +55,8 @@
   "app.components.PluginCard.price.free": "Gratuit",
   "app.components.PluginCard.more-details": "Plus de détails",
 
+  "app.utils.placeholder.defaultMessage": " ",
```

Comment:
Can we remove the space? And if not, can you put an insecable space instead?

Reply:
This will be fixed in See https://github.com/strapi/strapi/pull/564